frontend/src/pages/singleRecordPage/SingleRecordPage.js

```diff
@@ -0,0 +1,42 @@
+import React, {useContext} from 'react';
+
+import RecordContext from "../../contexts/RecordContext";
+import {useParams, useHistory } from 'react-router-dom';
+import Header from "../../commons/Header";
+import Record from "../../commons/Record";
+import RecordActionButton from "../../commons/RecordActionButton";
+
+export default function SingleRecordPage(){
+    const{records, deleteRecord} = useContext(RecordContext);
+
+    const {id} = useParams();
+    const history = useHistory();
+
+    const record = records.find((record) => record.id === id);
+    return !record ? null : (
+        <>
+            <Header titel={record.titel}/>
+            <Record record={record} actions={getActions()}/>
+        </>
+);
```

Comment:
for better readability I'd prefer the following:

```suggestion
    return { record &&
        <>
            <Header titel={record.titel}/>
            <Record record={record} actions={getActions()}/>
        </>
};
```